src/main/java/teammates/storage/search/SearchManagerFactory.java

```diff
@@ -25,4 +25,11 @@ public static void registerSearchManager(SearchManager searchManager) {
         }
     }
 
+    /**
+     * Deletes all search documents in the search manager instance.
+     */
+    public static void resetSearchManager() {
+        instance.resetCollections();
```

Comment:
As mentioned in previous review, the factory has no purpose having this method.

Reply:
You are right it seems out of place for this method to be here, and unnecessary too since there's already a `getSearchManager` method to access the `SeachManager` class.